app/jobs/application_job.rb

```diff
@@ -1,4 +1,15 @@
-# encoding: utf-8
+require 'newrelic_rpm'
 
 class ApplicationJob < ActiveJob::Base
+  around_perform :use_db_pool
+
+  rescue_from(StandardError) do |e|
```

Comment:
If I'm reading this correctly, this base class now surfaces all exceptions to New Relic? Nice!

Reply:
that's the idea